### DISCUSSION TITLE
How to get a github achievement?

### DISCUSSION THREAD

**Discussion by AnandaAlfa** [community/community#29650] [Other Feature Feedback, Questions, & Ideas] [answered]
Anyone hello? tell me a how to get that!

**Comment by mazkdevf** [comment]
https://github.com/Schweinepriester/github-profile-achievements

**Reply by mazkdevf** [reply]
check that out!

**Reply by AnandaAlfa** [reply]
thanks bro!

**Reply by mazkdevf** [reply]
> thanks bro!

np, have a good day!

**Comment by mndree** [comment, marked as answer]
you can see the task requirement in this link 

https://github.com/drknzz/GitHub-Achievements
